src/soilbiogeochem/SoilBiogeochemNitrogenFluxType.F90

```diff
@@ -1177,6 +1314,8 @@ subroutine Summary(this, bounds, num_soilc, filter_soilc)
        do fc = 1,num_soilc
           c = filter_soilc(fc)
           this%denit_col(c) = this%f_denit_col(c)
+ !mvm 06/19/2023 for coupling. Total soil N fluxes pass to the atmosphere
+            this%soil_nox_total_col(c)=this%f_nox_nit_atmos_col(c)+this%f_nox_denit_atmos_col(c) 
```

Comment:
should this flux be include `_atmos` somewhere in the name since it's actually the total soil NOx flux to the atmosphere?